### PR TITLE
Fix: Missing dependency of package rsync

### PR DESCRIPTION
**Pull request by Tronde**
## TL;DR

- The package 'rsync' is used by the module ansible.posix.synchronize
- Task 'extract_files | Extract files of mounted ISO to work directory' fails when package is missing
- Added 'rsync' as a dependency in the same way as 'xorriso'

## The epic explanaition

Hi Steffen,  
I tried your awesome role on a fresh install of RHEL 9.4 and discovered the following task failure do to the missing package `rsync`:

```
TASK [sscheib.rhel_iso_kickstart : extract_files | Extract files of mounted ISO to work directory: /home/jkastnin/downloads/src] ***
task path: /home/jkastnin/proj_rhel_iso_kickstart/roles/sscheib.rhel_iso_kickstart/tasks/extract_files.yml:31
…
    "msg": "Failed to find required executable \"rsync\" in paths: /sbin:/bin:/usr/sbin:/usr/bin:/usr/local/sbin"
```

According to `ansible-doc ansible.posix.synchronize` the module is only a wrapper for `rsync` that depends on the package to be present:

> NOTES:
>      * rsync must be installed on both the local and remote host.

To fix this I added `rsync` as a dependency to the role.

In case you have any further questions, you know where to find me.

### CODE REVIEW

**Review by sscheib — APPROVED** [review]

**Comment by sscheib** [comment]
Thanks for your contribution!

I assume you tested it on RHEL 9 prior to submitting the PR?

**Comment by Tronde** [comment]
> I assume you tested it on RHEL 9 prior to submitting the PR?

Yes, I ran it on a fresh RHEL 9.4 minimal installation  with `ansible-core`, the collection `ansible.posix` and this role installed.
